notebooks/region_search/Region Searching Workbook.ipynb

```diff
@@ -5,7 +5,7 @@
    "id": "d94d8d2b",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #7.        set_to_corners
</pre><p>It looks like this function is broken? If you want to leave it in as an interface, you could replace <code>set_to_corners</code> with <code>pass</code> for now  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/dirac-institute/kbmod/pull/461/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='94'/>